content/docs/getting-started/concepts.md

```diff
@@ -73,15 +73,18 @@ cf target -o ORGNAME -s SPACENAME
 
 All apps need to use a "buildpack" specific to their language, which sets up dependencies for their language stack. There are [standard buildpacks for most languages](https://docs.cloudfoundry.org/buildpacks/), and Cloud Foundry usually auto-detects and auto-applies the appropriate one when you deploy an app (one exception is that it doesn't auto-apply the special [binary buildpack](https://docs.cloudfoundry.org/buildpacks/binary/index.html)). We strongly encourage you to use the standard buildpacks. cloud.gov supports these standard buildpacks and provides security updates for them. (You'll need to redeploy or restage your application to pick up buildpack updates. You're also responsible for supporting and security-patching the application itself.)
 
-In the rare case where Cloud Foundry doesn't correctly auto-detect the buildpack, or if you want to use a binary buildpack or [custom buildpack]({{< relref "docs/apps/custom-buildpacks.md">}}), you can specify a buildpack in the [application manifest](http://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html) (as below) or with the `-b` flag. To reference it, use either the buildpack name:
```

Comment:
Why omit this? Seems like it's still needed after changing the syntax to fit with the `Or a URL:` a few lines down.

Reply:
I was confused by the usage of the word 'either' until you pointed out its significance to the URL section. I have re-introduced the sentence and tweaked the wording.